config/application.rb

```diff
@@ -12,6 +12,9 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.1
 
+    config.action_mailer.default_url_options = { host: ENV["RAILS_HOST"] }
+
     config.active_job.queue_adapter = :sidekiq
   end
 end
+Rails.application.routes.default_url_options[:host] = ENV["RAILS_HOST"]
```

Comment:
I made a ticket (#123) to add a default value to ansible, but it still seems like we're setting up a situation where if this value isn't set we're going to produce an error. Is this ever going to be anything except `localhost`? Can we set a default value? What if we did something like 
```
Rails.application.routes.default_url_options[:host] = ENV["RAILS_HOST"] || 'localhost'
```

Reply:
+1 I'll make it so